servers/rendering/renderer_rd/effects/vrs.cpp

```diff
@@ -34,7 +34,7 @@
 #include "../uniform_set_cache_rd.h"
 
 #ifndef _3D_DISABLED
-#include "servers/xr_server.h"
+#include "servers/xr/xr_server.h"
 #endif // _3D_DISABLED
```

Comment:
This should use `XR_DISABLED`, seems like it was missed in #103267, CC @YeldhamDev.